strapi/strapi
# add language names for Bosnian, Croatian, Macedonian and Serbian

### What does it do?

Adds language names for Bosnian, Croatian, Macedonian and Serbian to packages/core/admin/admin/src/translations/languageNativeNames.js

### Why is it needed?

Allow to extend translations for those languages in config.translation in src/admin/app.js

### How to test it?

Add some translations for the languages ba, hr, mk, rs to config.translation, build the admin and check the translations in the admin.
